doc/vector/code-samples/zvkg.h

```diff
@@ -29,4 +29,31 @@ zvkg_vghsh(
     const void* H
 );
 
+// Y, X, and H point to arrays of 128 bits values, 32b aligned
+// if the processor does not support unaligned access.
+// 'n' is the number 128b element groups.
+//
+//   Y <- (Y xor X) o H
+// Where 'o' is the Galois Field Multiplication.
+extern void
+zvkg_vghsh_vv(
+    void* Y,
+    const void* X,
+    const void* H,
+    size_t n
+);
+
+// X and Y point to arrays of 128 bits values, 32b aligned
+// if the processor does not support unaligned access.
+// 'n' is the number 128b element groups.
+//
+//   X <- (X xor Y)
```

Comment:
Shouldn't the formula actually use "o" ? (I am not against testing xor but I think it is less useful here :-) )
```suggestion
//   X <- (X o Y)
```

Reply:
Corrected.